### ISSUE TITLE
Is there an 'epoch' parameter in your training process.

### ISSUE THREAD

**Issue by chunchill** [Jzou44/Autoencoder-and-Oversampling-for-credit-card-fraud-detection-on-highly-imbalanced-dataset#1]
I have noticed that there are two ‘while True’ in the code without break clause, did you manually control the training process?

**Comment by chunchill** [comment]
BTW, why is the size of the test set is the same with the training set.
 self.test_feature = self.feature[train_dataset_size:, :]
 self.test_target = self.target[train_dataset_size:, :]
